src/simulator/simulator.cpp

```diff
@@ -169,9 +176,15 @@ void Simulator::loadObject() {
       new ShortTermObject("short_term_config.lua");
   objects.push_back(shortTermObject);
 
-  const vector<string> object_config_0 = {"config/human_config.lua"};
-  HumanObject* human_object0 = new HumanObject(object_config_0);
-  objects.push_back(human_object0);
+  // TODO (yifeng): load short term objects
+
+  // human
+  for (const string& config_str: CONFIG_human_config_list) {
+    const vector<string> object_config = {config_str};
+    HumanObject* human_object = new HumanObject(object_config);
+    objects.push_back(human_object);
```

Comment:
This currently leaks memory; the `objects` vector is currently of type

```
std::vector<EntityBase *> objects;
```

and should be changed to 

```
std::vector<std::unique_ptr<EntityBase>> objects;
```

and this code will have to be updated appropriately, along with the `ShortTermObject` loading code on the lines above.